openmls/src/treesync/treekem.rs

```diff
@@ -97,6 +97,13 @@ impl<'a> TreeSyncDiff<'a> {
         // ValSem202: Path must be the right length
         let direct_path_length = self.filtered_direct_path(params.sender_leaf_index).len();
         if direct_path_length != params.update_path.len() {
+            // XXX: Rewrite tests to allow for debug asserts.
```

Comment:
What do you mean here? Is this meant to be for a follow-up?

Reply:
Generally it would be nice to be able to write debug asserts. But we test a lot for errors and hence can't use debug asserts without rewriting the tests.
This one here I used for testing and just left in for future reference to use once we can. But if you prefer, I can remove it.